src/components/Settings/Settings.js

```diff
@@ -59,8 +59,8 @@ class Settings extends React.Component {
         }
     }
     componentDidMount(){
-        let state = this.state;
-        let url = BASE_URL+'/aaa/listUserSettings.json?'
+        var state = this.state;
```

Comment:
Any particular reason for this change?

Reply:
No, I was just trying out various changes and I forgot to revert this

Reply:
No issues, good work. :smile: